### PR TITLE
New IPC Antennae (and one more thing)

### PR DESCRIPTION
**Pull request by ParadoxSpace**
they're kind of like the anime antennae ears you see on androids and such.

image

image

image

image

also as i cannot help myself, vaurca clothes now have proper body_parts_covered

### CODE REVIEW

**Comment by ToasterStrudes** [comment]
thank you gods work

**Comment by ParadoxSpace** [comment]
!review

**Comment by miniusAreas** [comment]
@paradoxspace where are muh antler antenna

**Comment by Kaedwuff** [comment]
I don't really think that we should be implementing things just because they're anime references. The fact you included a shitty schoolgirl outfit in the reference is turning me off even more.

**Comment by ben10083** [comment]
Can we see an example of the vaurca ckothes thing?

**Comment by ToasterStrudes** [comment]
If I recall body_parts_covered affects what wounds are visible for each limb based on what the clothing covers, nothing really visual

**Comment by Arrow768** [comment]
Flagged as dnm until the other changes are reviewed aswell

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -11,6 +11,7 @@
 	icon_state = "harness"
 	worn_state = "harness"
 	item_state = "w_suit"
+	body_parts_covered = 0
```
Comment (MarinaGryphon) [inline code/modules/clothing/under/miscellaneous.dm]: I believe you should technically use `FALSE` or `null` here.

Comment (ParadoxSpace) [reply code/modules/clothing/under/miscellaneous.dm]: 0 is correct

**Comment by Arrow768** [comment]
@Kaedwuff Just because a antenna isnt used for radio communication doesnt mean its "useless".
There can be a number of reasons why there is a antenna attached to a IPC.

So I dont really have an issue with that PR.

_But I would welcome one that would remove that ugly uniform_

**Pull request merged**